x-pack/plugins/canvas/canvas_plugin_src/renderers/debug.tsx

```diff
@@ -26,9 +26,11 @@ export const debug: RendererFactory<any> = () => ({
 
     ReactDOM.render(renderDebug(), domNode, () => handlers.done());
 
-    handlers.onResize(() => {
-      ReactDOM.render(renderDebug(), domNode, () => handlers.done());
-    });
+    if (handlers.onResize) {
```

Comment:
@elastic/kibana-presentation this check was added to make it usable outside of canvas

Reply:
Seems pretty straightforward.